MIEngine.clrdbg.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.clrdbg.MIEngine$suffix$</id>
+    <version>1.0</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>MIEngine for use with clrdbg via VSCode</description>
+    <releaseNotes />
+    <copyright>Copyright 2015</copyright>
+  </metadata>
```

Comment:
I think you will eventually want to add a dependency here on OpenDebugAD7. But that can wait for another checkin...